nautobot/dcim/api/serializers.py

```diff
@@ -658,6 +698,23 @@ class Meta:
         extra_kwargs = {"cable": {"read_only": True}}
 
     def validate(self, data):
+        # set interface status to active on create (only!) if status was not provided
+        from nautobot.extras.models import Status
+
+        if self.instance is None and not data.get("status"):
+            # status is currently required in the Interface model but not required in api_version < 1.3 serializers
+            # which raises an error when validating except status is explicitly set here
+            query = Status.objects.get_for_model(Interface)
+            try:
+                data["status"] = query.get(slug=InterfaceStatusChoices.STATUS_ACTIVE)
+            except Status.DoesNotExist:
+                raise serializers.ValidationError(
+                    {
+                        "status": "Interface default status 'active' does not exist, "
+                        "create 'active' status for Interface or use the latest api_version"
+                    }
+                )
+
```

Comment:
Per the comment this is only relevant to API versions < 1.3, which we don't support in 2.x. Maybe this doesn't need to be added?
```suggestion
```